chromium_src/BUILD.gn

```diff
@@ -1186,3 +1187,14 @@ source_set("sessions") {
     "//third_party/protobuf:protobuf_lite",
   ]
 }
+
+source_set("net") {
```

Comment:
all chromium source sets should have 
```
configs += [ "//electron/build:electron_config" ]
configs += [ ":chromium_src_config" ]
```
to avoid potential build issues

Reply:
b9e3f89dd7a2cdbd0729006d1c969298a8624f01